en/flight_modes/flight_modes_summary.md

```diff
@@ -79,22 +79,44 @@ th {
 </tr>
 
 <tr>
- <td><a href="../flight_modes/altitude.md">Altitude</a>
+ <td><a href="../flight_modes/altitude_fw.md">Altitude</a>
  <p><a href="#key_difficulty"><img src="../../assets/site/difficulty_2.svg" title="Difficulty (Easy)" width="20px" /></a></p>
  </td>
  <td><p>S (roll)</p><p>S<sup>+</sup>(pitch)</p></td>
  <td>M</td>
  <td>S<sup>+</sup></td>
  <td><a href="#key_position_fixed"><img src="../../assets/site/position_fixed.svg" title="Altitude required (e.g. Baro, Rangefinder)" width="20px" /></a><sup><a href="#baro_only">++</a></sup></td>
- <td><p>This mode helps vehicle reach and maintain altitude.</p>
+ <td>
+ <!-- 
+ <p>This mode helps vehicle reach and maintain altitude.</p>
  <p>Centered RC RPY sticks gives level flight.</p>
  <p>Pitch input is used to control the altitude. If zero pitch input – autopilot holds current altitude against wind.</p>
  <p>Throttle stick controls the airspeed of the aircraft only if airspeed sensor is connected.</p>
  <p>Without airspeed sensor, the user cannot control Throttle.</p>
+ -->
+ <p>RC/manual mode like <a href="#stabilized_fw">Stabilized</a> mode but with <em>altitude stabilization</em> (centered sticks put vehicle into straight and level flight and maintain current altitude). The vehicle course is not maintained, and can drift due to wind.
+  <ul>
+    <li>Centered sticks (inside deadband):
+      <ul>
+       <li>Pitch input is used to control the altitude. If zero pitch input – autopilot holds current altitude against wind.</li> 
+       <li>RPY sticks gives level flight.</li> 
+       <li>Throttle stick controls the airspeed of the aircraft if an airspeed sensor is connected (without airspeed sensor, the user cannot control throttle).</li>
+    </ul>
+    <li>Outside center: <!-- (same as <em>Stabilized</em> mode?) -->
+      <ul>
```

Comment:
Is there anything we need to say for these? Specifically, my understanding is that pitch and roll are angle controlled. Upshot should mean that you can't roll over and you can't loop forwards - if so, shouldn't we mention that.